recipes-scanners/clamav/clamav_1.4.bb

```diff
@@ -0,0 +1,205 @@
+SUMMARY = "ClamAV anti-virus utility for Unix - command-line interface"
```

Comment:
I think this file needs to end with `_1.4.%.bb` to properly match the recipe `PV`. In fact, you should check that the value of `PV` is `1.4.3`.

`bitbake -e clamav | grep '^PV'`